server/php-engine.cpp

```diff
@@ -1664,6 +1667,7 @@ void init_all() {
     }
     log_server_warning(deprecation_warning);
   }
+  StatsHouseMetrics::get().init_common_tags(vk::singleton<ServerConfig>::get().get_cluster_name(), kdb_gethostname());
```

Comment:
ServerConfig will have mutable state in the future. For example get_cluster_name() function will be able to return a different value each time it is called.

Reply:
done, now we update common tags in cron